uaclient/api/tests/test_api_u_pro_token_info_v1.py

```diff
@@ -0,0 +1,48 @@
+import mock
+
+from uaclient.api.u.pro.token_info.v1 import (
+    AccountInfo,
+    ContractInfo,
+    ServiceInfo,
+    TokenInfoOptions,
+    TokenInfoResult,
+    _get_token_info,
+)
+from uaclient.cli.tests.test_cli_status import RESPONSE_CONTRACT_INFO
```

Comment:
eeek importing constants between tests smells
keep it if people don't mind - I mind a little bit

Reply:
Yeah, we can not import the constant.